### ISSUE TITLE
VA Forms System Core [Component]: Step 10 - ID attribute values must be unique

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#45718]
### Point of contact

David Kennedy (DK)

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

There are a few spots in the form where ID attributes aren't unique. These are:

```html
<section id="claimant-information">
<section id="veteran-information">
<section id="veteran-information-burial">
<section id="military-history-service-periods">
<section id="military-history-previous-names">
<section id="benefits-selection">
<section id="benefits-burial-allowance">
<section id="benefits-plot-allowance">
```

It looks like each of these has a h2 with the same ID. I'm not sure if this is an individual demo form problem with the implementation or something the form does on its own.

### Reproduction steps

1. View review page.
2. Search for the IDs above in the HTML, and note the multiple instances.

### Proposed solution or next steps

Only refer to an ID once.

### References, articles, or WCAG support

WCAG 4.1.1 Parsing

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [X] Axe-core
- [ ] Something else
